x/staking/keeper/keeper.go

```diff
@@ -147,7 +149,8 @@ func NewKeeper(
 			),
 			collections.BytesValue,
 		),
-		LastValidatorPower: collections.NewMap(sb, types.LastValidatorPowerKey, "last_validator_power", sdk.LengthPrefixedBytesKey, collections.BytesValue), // sdk.LengthPrefixedBytesKey is needed to retain state compatibility
+		// key format is: 17 | lengthPrefixedBytes(valAddr) | power
+		LastValidatorPower: collections.NewMap(sb, types.LastValidatorPowerKey, "last_validator_power", sdk.LengthPrefixedBytesKey, codec.CollValue[gogotypes.Int64Value](cdc)), // sdk.LengthPrefixedBytesKey is needed to retain state compatibility
```

Comment:
Changelog entry under state breaking change? cc: @julienrbrt

Reply:
I don't think it is state breaking.